templates/home/dashboard.html

```diff
@@ -165,7 +165,8 @@ <h4 class="text-center mt-3">Students Present</h4>
                         <tbody>
                             {% for stu_att in student_attendance %}
                             <tr class="text-center">
-                                <td class="text-monospace font-weight-lighter">{{stu_att.student.get_full_name}}</td>
+                                <a href="{% url 'students:profile' stu_att.student.id     %}"> <td class="text-monospace font-weight-lighter">
+                                    {{stu_att.student.get_full_name}}</td></a>
```

Comment:
Can you format it properly? Also, just use a single space between `id` and `%}`.